apps/shipments/models/shipment.py

```diff
@@ -13,23 +29,25 @@
 from django.contrib.gis.db.models import GeometryField
 from django.contrib.gis.geos import Point
 from django.contrib.postgres.fields import JSONField, ArrayField
-from django.core.validators import RegexValidator, MaxValueValidator, MinValueValidator
+from django.core.validators import RegexValidator, MinValueValidator
 from django.db import models
 from django_fsm import FSMIntegerField, transition
 from enumfields import Enum, EnumIntegerField
 from enumfields import EnumField
 from rest_framework.exceptions import Throttled, PermissionDenied
 from rest_framework.status import HTTP_200_OK, HTTP_503_SERVICE_UNAVAILABLE
 from influxdb_metrics.loader import log_metric
-from shipchain_common.utils import AliasField, random_id
+from shipchain_common.utils import random_id
 
 from apps.eth.fields import AddressField, HashField
 from apps.jobs.models import AsyncJob, JobState
 from apps.simple_history import TxmHistoricalRecords, AnonymousHistoricalMixin
-from .rpc import RPCClientFactory
+from ..rpc import RPCClientFactory
 
 LOG = logging.getLogger('transmission')
 
+# pylint: disable=too-many-branches
```

Comment:
What is triggering this pylint error? It doesn't seem like you've added any code to this file.

Reply:
Indeed I didn't add any new code here. I really don't know what triggers this.

Reply:
Can you please take another pass and see if you can remove this pylint suppressor? I am okay with suppressing pylint errors, but only if we understand the root cause and are willing to accept the technical debt.

Reply:
I was able to remove the `pylint` suppressor in `views.shipment` by code simplification , however here I wasn't able to solve the issue.